charts/connectors/testdata/template-cases.golden.txtar

```diff
@@ -10112,8 +10112,8 @@ spec:
         terminationMessagePath: /dev/termination-log
         terminationMessagePolicy: File
         volumeMounts:
-        - mountPath: /tmp
-          name: rp-connect-tmp
```

Comment:
Is this behavior correct? I'd imagine if you don't specify a volume mount in the `volumeMounts` part of the values file you'd just follow the default behavior if the volume itself still exists (in this case overridden in the values.yaml)?

It seems like if you override the volume we no longer delegate to the default mounting behavior with this change.